programs/install/Install.cpp

```diff
@@ -856,6 +856,10 @@ namespace
             {
                 fmt::print("The process with pid = {} is running.\n", pid);
             }
+            else
+            {
+                pid = 0;
```

Comment:
The code is incorrect:

man 2 kill:
```
RETURN VALUE
       On success (at least one signal was sent), zero is returned.  On error, -1 is returned, and errno is set appropriately.

ERRORS
       EINVAL An invalid signal was specified.

       EPERM  The calling process does not have permission to send the signal to any of the target processes.

       ESRCH  The target process or process group does not exist.  Note that an existing process might be a zombie, a process that has terminated execution, but has not yet been wait(2)ed for.
```

Reply:
You mean to check ESRCH explicitly?
I don't think there is any sense in that.
0 is not a real signal.

EINVAL responce is not possible. EPERM either (as no signal are actually send). 


It will just return 0 if process exists, and -1 otherwise. 
https://man7.org/linux/man-pages/man2/kill.2.html

>       If sig is 0, then no signal is sent, but existence and permission
>       checks are still performed; this can be used to check for the
>       existence of a process ID or process group ID that the caller is
>       permitted to signal.

Reply:
Ok i think EPERM is actually possible.